extension/storage/filestorage/Makefile

```diff
@@ -1 +0,0 @@
-include ../../Makefile.Common
```

Comment:
This is needed if filestorage is it's own module.

Reply:
Currently is not its own module. I went back and forth on whether it should be. Makefile was only here because it was left behind after removing other mod files.

Do you prefer it one way or the other?

Reply:
Oh, I did not know :). Rule is: If **NOT** a module **NO** Makefile